docs/usage/configuration-options.md

```diff
@@ -1120,6 +1120,19 @@ If you allow Renovate to run on "Selected" repositories, `forkProcessing` will b
 Only the `onboardingConfigFileName` (which defaults to `renovate.json`) is supported for `forkProcessing`.
 You can't use other filenames because Renovate only checks the default filename when using the Git-hosting platform's API.
 
+## gerritLabelMapping
```

Comment:
I would like to supersede this option with #23530 which will be more generic

Reply:
@rarkins what's the next step to get this PR moving again?

Reply:
https://github.com/renovatebot/renovate/pull/26047 was merged, so this `gerritLabelMapping` should be replaced with a guidance on how to use `statusCheckNames` instead

@sselberg maybe you can contribute this change?

Reply:
@rarkins as far as i understand the new `statusCheckNames` the setBranchStatus/getBranchStatus for each Platform will only be called if i've configured the correspinding statusCheckName for one of the checks?

Therefore i can remove my own `gerritLabelMapping` config and directly use the`context` Parameter from set/getBranchStatus as an Gerrit-Label?

Reply:
@NiasSt90 yes that's right. Please use statusCheckNames just like you were planning to use gerritLabelMapping. I guess you would want to populate default values if platform=gerrit?

Reply:
@rarkins  it's done. 

And i don't think we need default values for Gerrit here. 
Because `minimumReleaseAge` is not needed in most cases, because the default behavior of renovate delays the change-creation until `stabilityDays` are reached and `mergeConfidence` is a paid feature as far as i know (api key required?).